code/lib/core-common/src/index.ts

```diff
@@ -5,7 +5,6 @@ export * from './presets';
 export * from './utils/cache';
 export * from './utils/check-addon-order';
 export * from './utils/envs';
-export * from './utils/findDistEsm';
```

Comment:
If the file is no longer imported/exported, can it be removed?
Or is it used anywhere else?

Reply:
I did not find any use in the repository, so it has been moved to `code/frameworks/ember/src/util.ts`

Reply:
Ah, ok great, the file just moved. 👍